swagger.yaml

```diff
@@ -3105,6 +3105,19 @@ definitions:
               format: uuid
             hostname:
               type: string
+      selected-installation-disk-paths:
+        type: array
+        x-go-custom-tag: gorm:"type:varchar(64)[]"
+        description: The selected disk for installation.
+        x-nullable: true
+        items:
+          type: object
+          properties:
```

Comment:
to make things clear maybe you should add description to the fileds

Reply:
The description has been added to the tupple of (host-id, attribute)

Reply:
the (id, field) pairing pattern exists in 3 other `cluster-update-params` without any description, if a description is added here, it must be added to all of them

Reply:
I see not reason why, if it can help to make the api clearer then go for it.